### PR TITLE
Apply phpstan level 5

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes the following things:

- incorrect set error_reporting via ini_set, should be via `error_reporting` function
- ensure ini_set values to be string
- fix 4th parameter value to be 0 for preg_match instead of null
- ensure to use correct type in function values
- fix incorrect 2nd passed parameter of `lang()` function to be array.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

**Comment by samsonasik** [comment]
All green 🎉

File: system/Commands/Generators/CreateScaffold.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ public function run(array $params)
 		];
 
 		// Call those commands!
-		$class = $params[0] ?? CLI::getOption(0);
+		$class = $params[0] ?? CLI::getOption('0');
```
Comment (paulbalandan) [inline system/Commands/Generators/CreateScaffold.php:132]:
```suggestion
		$class = $params[0];
```
Good catch for this. But I think I made a bug here. this should be CLI::getArgument(0). Anyway, the params check is enough in this case.

File: system/Test/bootstrap.php [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?php
-ini_set('error_reporting', E_ALL);
+error_reporting (E_ALL);
```
Comment (paulbalandan) [inline system/Test/bootstrap.php:2]:
```suggestion
error_reporting(E_ALL);
```

File: system/Commands/Generators/CreateScaffold.php (hunk 1 above)
Comment (samsonasik) [reply system/Commands/Generators/CreateScaffold.php]: fixed.

File: system/Test/bootstrap.php (hunk 2 above)
Comment (samsonasik) [reply system/Test/bootstrap.php]: fixed.

**Pull request merged**